2016/proposals/prerit-garg-result-aggregation-server.md

```diff
@@ -0,0 +1,200 @@
+## Academic Experience 
+###About me
+* I am a final year B.Tech (Computer Engineering) student in Jamia Millia Islamia University, New Delhi, India. As a part of academic experience I am inclined towards Database, Data Structures and Algorithms.
+
+* My skill sets include : C/C++, Python - Flask, Django, Java, Android, HTML, CSS, JavaScript, MySQL, Sqlite, NoSQL - [MongoDB](https://www.mongodb.org/), REDIS, [ANSIBLE](https://www.ansible.com/).
+ 
+* I have mostly worked with Web Development ( RESTful API's, Dashboards etc ) while working as an intern with Zopper. I worked with Google Cloud Messaging ( GCM ) to send notification and bulk notification to more than 1 million zopper application users.  
+ 
+* I also have experience in Android Application Development. I developed an Android Application for [Litchi](http://www.litchi.co.in/), published it on PlayStore. and also a demo [FourSquare](https://github.com/prerit2010/FourSquare) application to gain more experience in android development. 
+ 
+* Apart from this, as the academic project I worked on cryptography, and published a research paper on "Efficient Substitution Box Design using Travelling Salesman Problem and Chaos". 
+ 
+## Why this project?
+I have a lot of experience in developing RESTful API's and web services. This project involves development of a RESTful API and setting up a server. Also, "Software Carpentry has been teaching researchers in science, engineering, medicine, and related disciplines the computing skills they need to get more done in less time and with less pain". This project is concerned with the problems that students face in installing the necessary packages for a workshop. This project will enable the instructors to better understand the problems and remove the package version and other issues, so that the setup process speeds up, and students don't get stuck in just installing the packages for a long time. Working upon this, it would give me more experience towards working on APIs and python Scripts, and also a open source experience which is very exciting for me.
+ 
+## Appendix
+###Code Contribution
+As per the requirement mentioned in the project information, I developed a sample [demo](https://github.com/prerit2010/DemoAPI) flask API which takes the Operating system information and sends to the API making a POST request.
+###Contact
+<table style="width:100%">
+  <tr>
+    <td>Name</td>
+    <td>Prerit Garg</td>
+  </tr>
+  <tr>
+    <td>Email Id</td>
+    <td>prerit2010@gmail.com</td>
+  </tr>
+  <tr>
+    <td>Github</td>
+    <td>https://github.com/prerit2010</td>
+  </tr>
+  <tr>
+    <td>Telephone</td>
+    <td>+91 9910883359</td>
```

Comment:
@prerit2010 We suggest that you remove your telephone number from here. **And use `rebase` to remove it from the Git history.**